kube_apiserver_metrics/datadog_checks/kube_apiserver_metrics/data/auto_conf.yaml

```diff
@@ -1,35 +1,44 @@
+## @param ad_identifiers - list of strings - required
+## A list of container identifiers that are used by Autodiscovery to identify
+## which container the check should be run against. For more information, see:
+## https://docs.datadoghq.com/agent/guide/ad_identifiers/
+#
 ad_identifiers:
   - kube-apiserver
 
+## All options defined here are available to all instances.
+#
 init_config:
 
+## Every instance is scheduled independent of the others.
+#
 instances:
+
     ## @param prometheus_url - string - required
     ## The URL where your application metrics are exposed by Prometheus.
     #
-  - prometheus_url: "https://%%host%%:6443/metrics"
+  - prometheus_url: https://%%host%%:6443/metrics
 
-    ## @param bearer_token_auth - string - optional
+    ## @param bearer_token_auth - boolean - optional - default: true
     ## Used if you are using RBACs and need the Agent to authenticate
     ## against the APIServer to retrieve metrics. Default to true.
     #
-    bearer_token_auth: true
+    # bearer_token_auth: true
```

Comment:
Just to be sure.

So the default is false in the mixins: https://github.com/DataDog/integrations-core/blob/b0e7f0a56f7569aef8a10d932d1027203ebd0647/datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py#L309

But `kube_apiserver_metrics` integration is overriding it ? https://github.com/DataDog/integrations-core/blob/de6584755a95fb6b5857f540e66c73f94d1434ba/kube_apiserver_metrics/datadog_checks/kube_apiserver_metrics/kube_apiserver_metrics.py#L100-L101

Not sure if we should or not still keep `bearer_token_auth: true` to make it explicit.

Reply:
it is explicit cause it's stated that that's the default value